ricecooker/classes/questions.py

```diff
@@ -181,7 +181,7 @@ def set_image(self, text):
         filename = exercise_file.process_file()
 
         # Need to replace text's web+graphie with https to get text matches
-        text = text.replace("web+graphie://", "https://").replace(path_text, exercises.CONTENT_STORAGE_FORMAT.format(exercise_file.get_replacement_str()))
+        text.replace("web+graphie://", "web+graphie:https://").replace(path_text, exercises.CONTENT_STORAGE_FORMAT.format(exercise_file.get_replacement_str()))
```

Comment:
Might need to change it to "text = text.replace..." otherwise the text won't get set to the new value

Reply:
whoops!